src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCountTableColumns.java

```diff
@@ -36,7 +36,7 @@
     public static final List<String> FULL_COLUMN_NAME_ARRAY = createUnmodifiableList(
             CONTIG, POSITION, REF_COUNT, ALT_COUNT, REF_NUCLEOTIDE, ALT_NUCLEOTIDE, READ_DEPTH, HET_LOG_ODDS);
 
-    public static List<String> getColumns(final AllelicCountTableVerbosity verbosity) {
+    public static List<String> getColumns(final AllelicCountTableVerbosity verbosity) throws UserException.BadInput {
```

Comment:
You don't need to add unchecked exceptions to the throws list, these are implied. Is better to document them in the Java-doc.

Not important, just change this if you are going to edit the file for some reason.